### PR TITLE
PIM-5900: Fix table borders and disabled buttons, remove upload-and-import button

### PR DESCRIPTION
**Pull request by pierallard**

### CODE REVIEW

**Comment by loicguillois** [comment]
overwriting bootstrap was a bad idea :( but it's legacy now I supposed ... so GTM

File: features/Context/WebUser.php [hunk 1]
```diff
@@ -1314,15 +1314,22 @@ public function iPressTheButton($button)
...
     /**
-     * @param string $button
+     * @param string      $button
+     * @param string|null $disabled
      *
-     * @Given /^I should see the "([^"]*)" button$/
+     * @Given /^I should see the "([^"]*)"( disabled)? button$/
      */
-    public function iShouldSeeTheButton($button)
+    public function iShouldSeeTheButton($button, $disabled = null)
```
Comment (pierallard) [inline features/Context/WebUser.php]: bool?

File: features/Context/WebUser.php [hunk 2]
```diff
@@ -1314,15 +1314,22 @@ public function iPressTheButton($button)
...
-        $this->getMainContext()->spin(function () use ($button) {
+        $buttonNode = $this->getMainContext()->spin(function () use ($button) {
             return $this->getCurrentPage()->getButton($button);
         }, sprintf("Can not find any '%s' button", $button));
+
+        if (null !== $disabled) {
+            $this->spin(function () use ($buttonNode) {
+                return $buttonNode->hasClass('disabled');
+            }, sprintf("The button '%s' is not disabled", $button));
+        }
```
Comment (pierallard) [inline features/Context/WebUser.php]: refaco

**Comment by aRn0D** [comment]
GTM once fixed

File: features/Context/WebUser.php [hunk 3]
```diff
@@ -1314,15 +1314,26 @@ public function iPressTheButton($button)
     }
 
     /**
-     * @param string $button
+     * @param string      $button
+     * @param string|bool $disabled
      *
-     * @Given /^I should see the "([^"]*)" button$/
+     * @Given /^I should see the "([^"]*)"( disabled)? button$/
```
Comment (phaseinducer) [inline features/Context/WebUser.php]: Not the same step at all imho.
In one case you test if a button is here, and in the other you test if a button is disabled.

**Comment by phaseinducer** [comment]
Small comment then GTM

**Pull request merged**